globaldog.py

```diff
@@ -63,7 +65,7 @@ def handleErrors(datadogResponse):
 screenboard["board_title"] = "{} {}".format(
```

Comment:
```python
print("Sending updated screenboard to Datadog...")
```

It looks like the change to "pint" is likely a typo. This change would cause a runtime error since "pint" is not defined. Reverting it to "print" maintains the function as intended.